libpod/container_internal_linux.go

```diff
@@ -664,7 +664,7 @@ func (c *Container) makePlatformBindMounts() error {
 	// Make /etc/hostname
 	// This should never change, so no need to recreate if it exists
 	if _, ok := c.state.BindMounts["/etc/hostname"]; !ok {
-		hostnamePath, err := c.writeStringToRundir("hostname", c.Hostname())
+		hostnamePath, err := c.writeStringToRundir("hostname", fmt.Sprintf("%s\n", c.Hostname()))
```

Comment:
there isn't really a strong reason for sprintf here, you could do `c.Hostname() + "\n"` instead

Reply:
updated

Reply:
Thanks, looks like the linter wants the spaces removed `c.Hostname()+"\n"` though

Reply:
thx, updated